astropy/modeling/blackbody.py

```diff
@@ -170,7 +170,7 @@ class BlackBody1D(Fittable1DModel):
     # bolometric flux is the integral of the model over the spectral axis. This
     # is more useful than simply having an amplitude parameter.
     temperature = Parameter(default=5000, min=0, unit=u.K)
-    bolometric_flux = Parameter(default=1, unit=u.erg / u.cm ** 2 / u.s)
+    bolometric_flux = Parameter(default=1, min=0, unit=u.erg / u.cm ** 2 / u.s)
```

Comment:
This only makes sense if we will never support magnitude units for this parameter. Currently, setting `bolometric_flux` to magnitude does not raise an exception on init but it chokes on evaluation. What do you think, @astrofrog ?

Reply:
@pllim Are you saying if the model is initialized with units `evaluate` does not work? If so can you post an example, perhaps create an issue?

Reply:
When flux is set to magnitude, yes, it does not work. However, I don't know if flux in magnitude should even be supported, that's why I didn't create an issue. Magnitude support can get tricky real fast.

Reply:
Saw this only as it drifted up top because of being opened and closed. FWIW, I think we should worry about magnitudes when they can actually be used.